test/indexing_after_ns_removal.sh

```
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# This is a test suite for checking the index mapping after the project has
+# been deleted and another namespaces has been created with the same name. 
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+os::test::junit::declare_suite_start "test/indexing_after_ns_removal"
+
+
+create_pod(){
+# keep indentation clean
+    oc create -f - <<EOF &>/dev/null
+apiVersion: v1
+kind: Pod
+metadata:
+  name: bar
+  namespace: $NS
+spec:
+  containers:
+  - image: gcr.io/google_containers/busybox:latest
+    name: bar
+    command: [ "/bin/sh", "-c", "echo foo-bar $1" ]
+  restartPolicy: Never
+EOF
+}
+
+
+cleanup(){
+    local return_code="$?"
+
+    for index in $(oc rsh -c elasticsearch $es_pod es_util  --query=_cat/indices?h=index,docs.count | grep pelle | awk '{ print $1 }')
+    do 
+        oc exec -c elasticsearch $es_pod -- curl --key /etc/elasticsearch/secret/admin-key   --cert /etc/elasticsearch/secret/admin-cert   --cacert /etc/elasticsearch/secret/admin-ca -XDELETE   "https://localhost:9200/$index"
+    done
```

```
curl_es $es_pod /*pelle* -XDELETE || :
```
That is, delete all indices matching `*pelle*`, and ignore errors

---

and
```
curl_es $es_pod /_cat/indices 2>&1 | artifact_out
```
to dump the indices before deleting them